src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java

```diff
@@ -157,6 +157,16 @@
     )
     private WhoDivorcing divorceWho;
 
+    @CCD(
+        label = "The applicant wants to continue with their application."
+    )
+    private YesOrNo continueApplication;
+
+    @CCD(
+        label = "Has the applicant been sent a reminder to apply for the Final Order?"
+    )
+    private YesOrNo finalOrderReminderSent;
```

Comment:
should this be part of FinalOrder class ?

Reply:
I added it to the Applicant class as I thought that a final order reminder could be sent to the Applicant and the respondent.

Reply:
checked with Linus it should be in FinalOrder class

Reply:
ok

Reply:
yeah, it'll need two, one for each applicant.

Reply:
Updated name from finalOrderReminderSent to finalOrderReminderSentApplicant2 and added the ...Applicant1 field for when it is required